controller/apps.go

```diff
@@ -0,0 +1,413 @@
+package controller
+
+import (
+	"context"
+	"encoding/json"
+	"strconv"
+	"time"
+
+	"github.com/fabric8-services/fabric8-wit/app"
+	"github.com/fabric8-services/fabric8-wit/configuration"
+	"github.com/fabric8-services/fabric8-wit/errors"
+	"github.com/goadesign/goa"
+	uuid "github.com/satori/go.uuid"
+)
+
+// AppsController implements the apps resource.
+type AppsController struct {
+	*goa.Controller
+	AuthURL string
+	WitURL  string
+}
+
+// NewAppsController creates a apps controller.
+func NewAppsController(service *goa.Service, config *configuration.Registry) *AppsController {
```

Comment:
all endpoint methods of this controller may need to include some authorization checking to limit access to the data to the namespace owner/collborator, I believe

Reply:
Please see my reply to your comment on design/apps.go.